.github/workflows/src/context.js

```diff
@@ -210,6 +210,36 @@ export async function extractInputs(github, context, core) {
       issue_number: issue_number,
       run_id: payload.workflow_run.id,
     };
+  } else if (context.eventName === "check_run") {
```

Comment:
```suggestion
  } else if (context.eventName === "check_run" && context.payload.action === "completed") {
```

I'm not sure this is the exact property name, but I think this should be scoped to only "completed" events, until we need otherwise.

Reply:
Leave it to handle by each individual workflow template.